src/rules/order.js

```diff
@@ -240,17 +241,61 @@ function makeOutOfOrderReport(context, imported) {
   reportOutOfOrder(context, imported, outOfOrder, 'before')
 }
 
+function mutateRanksToAlphabetize(imported, order, ignoreCase) {
+  const groupedByRanks = imported.reduce(function(acc, importedItem) {
+    acc[importedItem.rank] = acc[importedItem.rank] || []
+    acc[importedItem.rank].push(importedItem.name)
+    return acc
+  }, {})
+
+  const groupRanks = Object.keys(groupedByRanks)
+
+  // sort imports locally within their group
+  groupRanks.forEach(function(groupRank) {
+    groupedByRanks[groupRank].sort(function(importA, importB) {
+      return ignoreCase
+        ? importA.localeCompare(importB)
+        : (importA < importB ? -1 : (importA === importB ? 0 : 1))
```

Comment:
Scary. Could it be replaced with `if... return`?

Reply:
Btw it is also incorrect I think. The default `sensitvity` option is `variant` in `localeCompare` for the default `sort`ing usage (applies in the above snippet), which [does not ignore casing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#Parameters).

I don't want to meddle in this PR, but I suggest the following:
```
groupedByRanks[groupRank].sort(function(importA, importB) {
  return importA.localeCompare(importB, undefined, { sensitivity: ignoreCase ? 'accent' : 'case' })
})
```